src/openzaak/management/commands/setup_configuration.py

```diff
@@ -0,0 +1,104 @@
+        parser.add_argument(
+            "openzaak_domain",
+            help="Specifies the domain for this Open Zaak installation",
+        )
+        parser.add_argument(
+            "notifications_api_root",
+            help="Specifies the API root for the notifications API.",
+        )
+        parser.add_argument(
+            "municipality",
+            help="Name of the municipality to which the Open Zaak installation belongs",
+        )
+        parser.add_argument(
+            "openzaak_to_notif_secret",
+            help="Secret used for the Application that allows Open Zaak to retrieve notifications",
+        )
+        parser.add_argument(
+            "notif_to_openzaak_secret",
+            help="Secret used for the Application that allows Notifications API to retrieve authorizations",
+        )
+
+    @transaction.atomic
+    def handle(self, *args, **options):
+        try:
+            Applicatie.objects.all().delete()
+            openzaak_domain = options["openzaak_domain"]
+            notifications_api_root = options["notifications_api_root"]
+            municipality = options["municipality"]
+            openzaak_to_notif_secret = options["openzaak_to_notif_secret"]
+            notif_to_openzaak_secret = options["notif_to_openzaak_secret"]
+
+            site = Site.objects.get_current()
+            site.domain = openzaak_domain
+            site.name = f"Open Zaak {municipality.capitalize()}"
+            site.save()
+
+            notif_config = NotificationsConfig.get_solo()
+            notif_config.api_root = notifications_api_root
+            notif_config.save()
+
+            if not APICredential.objects.filter(api_root=notifications_api_root).exists():
+                APICredential.objects.create(
+                    api_root=notifications_api_root,
+                    label=f"Open Notificaties {municipality.capitalize()}",
+                    client_id=f"open-zaak-{municipality}",
+                    secret=openzaak_to_notif_secret,
+                    user_id=f"open-zaak-{municipality}",
+                    user_representation=f"Open Zaak {municipality.capitalize()}",
+                )
+
+            notif_api_jwtsecret_ac = JWTSecret.objects.create(
+                identifier=f"open-notificaties-{municipality}",
+                secret=notif_to_openzaak_secret,
+            )
+            notif_api_applicatie_ac = Applicatie.objects.create(
+                label=f"Open Notificaties {municipality.capitalize()}",
+                client_ids=[notif_api_jwtsecret_ac.identifier],
+            )
+            Autorisatie.objects.create(
+                applicatie=notif_api_applicatie_ac,
+                component=ComponentTypes.ac,
+                scopes=[SCOPE_AUTORISATIES_LEZEN],
+            )
+
+            openzaak_applicatie_notif = Applicatie.objects.create(
+                label=f"Open Zaak {municipality.capitalize()}",
+                client_ids=[f"open-zaak-{municipality}"],
```

Comment:
@joeribekker this is step #4 under `Open Zaak` [here](https://open-zaak.readthedocs.io/en/latest/installation/configuration.html). In the manual it specifies that I can use any secret and that it doesn't need to be used in any other step, but for this application (that gives OZ access to notificaties) to work, the clientid/secret must also be registered in Open notificaties right?

Here I used the same clientid/secret as used by the external API credential that for Open Zaak -> Open Notificaties (step #2 in the manual), not sure if that's right